nautobot-app/{{ cookiecutter.project_slug }}/docs/dev/contributing.md

```diff
@@ -53,20 +53,26 @@ The branching policy includes the following tenets:
 
 {{ cookiecutter.verbose_name }} has currently no intended scheduled release schedule, and will release new features in minor versions.
 
-When a new release, from `develop` to `main`, is created the following should happen.
+When a release is ready to be created from either `develop` or `ltm-x.x`, the following should happen.
 
-- A release PR is created from `develop` with:
+- Create a release PR by:
+    - Source from `develop` or `ltm-<major>.<minor>` branch and creatch new branch, generally `release/<major>.<minor>.<patch>`.
     - Update the release notes in `docs/admin/release_notes/version_<major>.<minor>.md` file to reflect the changes.
+        - You can run `invoke generate-release-notes` to generate these notes and delete the legacy towncrier fragments.
+        - Please consider adding changelog's from ltm releases in current release, as applicable.
+    - Update the mkdocs.yml file to include the reference to `docs/admin/release_notes/version_<major>.<minor>.md` as applicable.
     - Change the version from `<major>.<minor>.<patch>-beta` to `<major>.<minor>.<patch>` in `pyproject.toml`.
```

Comment:
Best way to do this is `poetry version <major>.<minor>.<patch>` rather than manually editing pyproject.toml. Same comment on line 70/76 below.

Reply:
(or to just bump it to the next release version, `poetry version patch` or `poetry version minor` as appropriate).